### DISCUSSION TITLE
any place i can learn to use the command line?

### DISCUSSION THREAD

**Discussion by tjquan91** [Chia-Network/chia-blockchain#1694] [Q&A]
Hi, im now plotting and farming on the GUI and have zero experience using command lines. 

Im going to use an extra machine to do the plotting (which requires use of command line).

Could anyone please give me a direction on how to do that? 
Like i dont even know where to type in the commands

**Comment by KrisMie** [comment]
Hi.
Give us more information, i.e. what operating system, do you have monitor connected to it or are you going to connect over the network?
Normally if you have no GUI linux machine it starts to command line ready for you to log in and work. If its machine with GUI you have to start command line terminal by youreself. Research how to do that in your system.
Some quick resources:
https://www.computerhope.com/issues/chusedos.htm
https://ubuntu.com/tutorials/command-line-for-beginners#1-overview
If you want to connect to terminal remotely:
https://phoenixnap.com/kb/ssh-to-connect-to-remote-server-linux-or-windows
Enjoy the lecture.
